Products/PleiadesEntity/skins/PleiadesEntity/place_view.pt

```diff
@@ -118,6 +118,15 @@
         <p tal:condition="not:portal_state/anonymous">Red titles indicate drafting-state items. Yellow titles indicate pending-state.</p>
 
         <div tal:condition="not:context/@@iterate/baseline|nothing">
+
+          <div id="representativePointField" class="field">
+            <label>Representative Point:</label>
+            <dl tal:replace="structure context/@@representative-point"></dl>
+            <div tal:condition="not:portal_state/anonymous">
```

Comment:
yes that line seems ineed pointless, I copied it from the locationsField (line 134), I guess without thinking too much about it :(
ok, I see what you mean about the baseline sections. Actually I wonder if this field which is strictly for view purposes (ie not a widget for adding/updating), it should not be factored out of the baseline condition blocks entirely, to say line 116?